internal/stack/config.go

```diff
@@ -52,14 +52,19 @@ func defaultConfig(profile *profile.Profile) Config {
 		ElasticsearchUsername: elasticsearchUsername,
 		ElasticsearchPassword: elasticsearchPassword,
 		KibanaHost:            "https://127.0.0.1:5601",
-		CACertFile:            profile.Path(CACertificateFile),
 	}
 }
 
 func LoadConfig(profile *profile.Profile) (Config, error) {
 	d, err := os.ReadFile(configPath(profile))
 	if errors.Is(err, os.ErrNotExist) {
-		return defaultConfig(profile), nil
+		config := defaultConfig()
+		caCertFile := profile.Path(CACertificateFile)
+		// Use CA file in the profile only if it exists.
+		if _, err := os.Stat(caCertFile); err == nil {
+			config.CACertFile = caCertFile
```

Comment:
This is likely never going to happen. This can only happen with data directories created with some very old versions of elastic-package, that didn't have profile config, but had certificates.

It would be probably fine to just don't set the CA cert file, keeping it just in case.